### PR TITLE
Docs: fix gotk create helmrelease examples

### PR DESCRIPTION
**Pull request by scottrigby**
Issues:
- `✗ unknown flag: --chart-name`
- `✗ chart name or path is required`
- `✗ source must be in format <kind>/<name>`
- `✗ semver range parse error: Could not parse Range "^1.10.0": Could not parse comparator "^" in "^1.10.0"`

### CODE REVIEW

File: docs/guides/installation.md [hunk 1]
```diff
@@ -324,9 +324,9 @@ gotk create helmrelease sealed-secrets \
 --interval=1h \
 --release-name=sealed-secrets \
 --target-namespace=gitops-system \
---source=stable \
---chart-name=sealed-secrets \
---chart-version="^1.10.0"
+--source=HelmRepository/stable \
+--chart=sealed-secrets \
+--chart-version="1.10.3"
```
Comment (hiddeco) [inline docs/guides/installation.md:329]:
```suggestion
--chart-version="1.10.x"
```

is likely more fitting to subscribe to patch updates.

File: docs/guides/sealed-secrets.md [hunk 2]
```diff
@@ -50,13 +50,13 @@ gotk create helmrelease sealed-secrets \
 --interval=1h \
 --release-name=sealed-secrets \
 --target-namespace=gitops-system \
---source=stable \
---chart-name=sealed-secrets \
---chart-version="^1.10.0"
+--source=HelmRepository/stable \
+--chart=sealed-secrets \
+--chart-version="1.10.3"
```
Comment (hiddeco) [inline docs/guides/sealed-secrets.md:55]:
```suggestion
--chart-version="1.10.x"
```

(same as the above)

File: docs/guides/sealed-secrets.md [hunk 3]
```diff
@@ -50,13 +50,13 @@ gotk create helmrelease sealed-secrets \
...
 ```
 
-With chart version `^1.10.0` we configure helm-controller to automatically upgrade the release
-when a new chart version is fetch by source-controller. 
+With chart version `1.10.3` we configure helm-controller to automatically upgrade the release
+when a new chart version is fetch by source-controller.
```
Comment (hiddeco) [inline docs/guides/sealed-secrets.md:59]:
```suggestion
With chart version `1.10.x` we configure helm-controller to automatically upgrade the release
when a new chart patch version is fetch by source-controller.
```

**Review by hiddeco — APPROVED** [review]
Perfect, thank you @scottrigby :tulip: 

(Squashed the commits for you)

**Comment by scottrigby** [comment]
Yeah I was going to ask if you have the squash and merge option configured for this repo 🙂

**Comment by hiddeco** [comment]
@scottrigby we use merge commits, and (ask people to) squash the commits by hand.

**Pull request merged**